clang-tools-extra/docs/ReleaseNotes.rst

```diff
@@ -498,6 +498,10 @@ Changes in existing checks
   false-positives when type of the member does not match the type of the
   initializer.
 
+- Improved :doc:`readability-redundant-smartptr-get
```

Comment:
you may need to rebase, as release notes has been cleared after branch out

Reply:
will deleting the old branch work?

Reply:
I cannot see those markers in my local branch